src/App.tsx

```diff
@@ -47,7 +47,7 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 export const App: React.FC = () => {
     const classes = useStyles();
-    const history = useHistory();
+    const navigation = useNavigate();
```

Comment:
I would call this variable `navigate` instead so it reads more like a verb when you call the function below.

Reply:
We already have navigate defined as a function at line 58